### PR TITLE
Add Grpc function to get root bounding cube

### PR DESCRIPTION
**Pull request by thomasschiwietz**
Fixes part of #86

### CODE REVIEW

File: grpc/src/proto.proto [hunk 1]
```diff
@@ -25,10 +25,18 @@ service Octree {
   // TODO(sirver): This is not the most efficient interface for a gRPC
   // version. We probably just send the node metadata to the client and let
   // the client do figure out GetVisibleNodes.
+  rpc GetRootBoundingCube(Empty) returns (GetRootBoundingCubeReply);
   rpc GetVisibleNodes(GetVisibleNodesRequest) returns (GetVisibleNodesReply);
   rpc GetNodeData(GetNodeDataRequest) returns (GetNodeDataReply);
 }
 
+message Empty {
```
Comment (SirVer) [inline grpc/src/proto.proto]: Just call it GetRootBoundingCubeRequest. We might want to add stuff there in the future.

File: grpc/src/bin/octree_server.rs [hunk 2]
```diff
@@ -125,7 +150,7 @@ fn main() {
     let service = proto_grpc::create_octree(OctreeService { octree });
     let mut server = ServerBuilder::new(env)
         .register_service(service)
-        .bind("" /* ip to bind to */, port)
+        .bind("127.0.0.1" /* ip to bind to */, port)            // TODO(tschiwietz): removed hardcoded localhost
```
Comment (SirVer) [inline grpc/src/bin/octree_server.rs]: try 0.0.0.0 - that is used in the gRPC test suite.

File: grpc/src/bin/octree_server.rs [hunk 3]
```diff
@@ -26,6 +27,30 @@ struct OctreeService {
 }
 
 impl proto_grpc::Octree for OctreeService {
+    fn get_root_bounding_cube(
```
Comment (SirVer) [inline grpc/src/bin/octree_server.rs]: did you run g fix?

**Comment by SirVer** [comment]
you have conflicts, otherwise lgtm.

**Comment by SirVer** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user SirVer. Merge queue now has a length of 1.

**Pull request merged**